lib/elastic_searcher.rb

```diff
@@ -65,6 +74,16 @@ def search_definition # rubocop:disable Metrics/MethodLength
     end
   end
 
+  def suggestions_definition
+    query_str = @query
+    source_array = @api ? api_source : ui_source
```

Comment:
Same as above.. no need for specific handling for `@api`. `ui_source` or just `name` as the source would work.

Reply:
Done this in commit `Pass api as an argument for search`.